apps/namadillo/src/App/AccountOverview/UnshieldedAssetsOverview.tsx

```diff
@@ -0,0 +1,64 @@
+import { ActionButton, Panel, SkeletonLoading } from "@namada/components";
+import { routes } from "App/routes";
+import { transparentTokensAtom } from "atoms/balance";
+import { useAmountsInFiat } from "hooks/useAmountsInFiat";
+import { useAtomValue } from "jotai";
+import { useLocation, useNavigate } from "react-router-dom";
+import { StakeYourNamCard } from "./StakeYourNamCard";
+import { TotalBalanceCard } from "./TotalBalanceCard";
+import { UnshieldedAssetTable } from "./UnshieldedAssetTable";
+
```

Comment:
<img width="182" alt="Screenshot 2025-04-01 at 5 47 23 PM" src="https://github.com/user-attachments/assets/b9e0fdc5-80b1-4477-bd1d-c5b1573f5b4c" />


Can we make these two icons have a tooltip?

It's not really clear to users what they do without clicking them and navigating away to a new page.

Reply:
@ChrisHoltDesign what do you think? Maybe change it to text directly, like "Shield"? 
This is also related with the new Transfer area which could have different actions and drop (make it invisible for the user) things like 'Ibc transfer' etc

Reply:
<img width="1186" alt="Screenshot 2025-04-08 at 8 48 49 AM" src="https://github.com/user-attachments/assets/f08fc5c9-ff21-4632-b5aa-e7cebc44b266" />